commands/commands.go

```diff
@@ -636,3 +645,19 @@ func getCertPathInfo(c *cli.Context) libmachine.CertPathInfo {
 		ClientKeyPath:  clientKeyPath,
 	}
 }
+
+func detectShell() (string, error) {
+	// attempt to get the SHELL env var
+	shell := filepath.Base(os.Getenv("SHELL"))
```

Comment:
Is this wrapped in `filepath.Base` so that we get the value we expect in case SHELL contains something like `/usr/bin/zsh`?  Just making sure I understand correctly.